README.md

```diff
@@ -241,27 +272,6 @@ push:
 
 See the [push-on-changes.yaml](examples/push-on-changes/buf-ci.yaml) example.
 
-### Verify generated files are up-to-date
-
-If your project uses local code generation, we recommend checking for diffs on pull requests.
-This isn't available as a built-in step because generating code may require manual setup, but can easily be added by invoking `buf`.
-
-To check that generated files match committed protobuf files, run the `buf generate` command and then `git diff`.
-If differences exist, `git diff` returns a non-zero exit code with the `--exit-code` flag.
-
-```yaml
-- name: Run buf generate
-  run: |
-    buf generate --error-format github-actions
-    git diff --exit-code gen
-```
-
-#### Builtin protoc plugins
-
-Some projects require the use of builtin `protoc` plugins, such as `protoc-gen-cpp`.
-To use these plugins, please additionaly install `protoc` such as with the action
-[`setup-protoc`](https://github.com/marketplace/actions/setup-protoc).
-
 ### Example workflows
 
```

Comment:
Did the migration step need to be removed? https://github.com/bufbuild/buf-action/pull/37/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R279

Reply:
I don't remember any discussion of removing that.